Spec/TestUtilities.swift

```diff
@@ -1506,5 +1506,32 @@ extension ARTHTTPPaginatedResponse {
     var headers: NSDictionary {
         return response.objc_allHeaderFields
     }
+}
+
+protocol ARTHasInternal {
+    associatedtype Internal
+    func unwrapAsync(_: @escaping (Internal) -> ())
+}
+
+extension ARTHasInternal {
+    var _in: Internal {
```

Comment:
Where is this necessary?

Reply:
Originally I used this throughout the test suite but gave it up. This was left over.

Removed at 0a757fe.